website/source/docs/agent/options.html.md

```diff
@@ -301,8 +301,9 @@ The options below are all specified on the command-line.
 
 <a name="_retry_join"></a>
 
-* `-retry-join` - Similar to [`-join`](#_join) but allows retrying a join if the
-  first attempt fails. This is useful for cases where you know the address will
+* `-retry-join` - Similar to [`-join`](#_join) but allows retrying a join until
+  it is successful. When it is joined the first time it will never attempt
+  to join again. This is useful for cases where you know the address will
```

Comment:
Perhaps we should explain that it will never attempt to join this list of peers again but a disconnected node will still rejoin the cluster based on all the nodes it knew about before?

```suggestion
  to join again. Agents will then maintain their membership based on the peers they discover via gossip. This is useful for cases where you know the address will
```

Reply:
Iterating on this (let me know if too redundant)  "Once it joins successfully to a member in a list of members it will never attempt to join again. Agents will then solely maintain their membership via gossip"

Reply:
Sounds great!